src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+    public String doWork() {
+        return "Bulldozer started its work.";
+    }
+
+    @Override
+    public String stopWork() {
+        return "Bulldozer stopped its work.";
+    }

```

Comment:
![image](https://user-images.githubusercontent.com/20326858/199470401-b9d7b50c-306e-45bf-986c-c30d56647c6e.png)
The task is to PRINT in these methods. Make them void, do not return String.